charts/gardener-extension-provider-alicloud/values.yaml

```diff
@@ -58,6 +58,7 @@ config:
     storage:
       className: gardener.cloud-fast
       capacity: 25Gi
+#  mutateKubeAPIServerService: true
```

Comment:
What about a cleaner structure here, i.e.
```suggestion
# kubeAPIServer:
#   mutateExternalTrafficPolicy: true
```

Reply:
OK. Also changed the config struct.